zcash_primitives/src/transaction/mod.rs

```diff
@@ -593,6 +729,11 @@ impl Transaction {
         Transaction { txid, data }
     }
 
+    #[cfg(zcash_unstable = "nu6")] /* TODO nu7 */
+    fn from_data_v7(data: TransactionData<Authorized>) -> Self {
+        Self::from_data_v5(data)
```

Comment:
this doesn't seems right. we have different bundles for `v6`. is it really what meant?

Reply:
Bundles are different but in "separate-bundles" concept they are both optionally inside TransactionData so the flow is the same, just different fields are present inside TransactionData